setup.py

```diff
@@ -347,10 +344,10 @@ def get_config_var(name):
                                                      remove='lib/cartopy/')) +\
                              list(file_walk_relative('lib/cartopy/data/shapefiles/gshhs',
                                                      remove='lib/cartopy/')) +\
-                             ['io/srtm.json']
+                             ['io/srtm.json'],
+                  '': ['requirements/*.txt'],
```

Comment:
The requirements weren't being included as part of the distribution, and therefore couldn't be used by PIP when installing.